source/includes/webhooks/_candidate_events.md

```diff
@@ -65,7 +65,7 @@ This web hook will fire when a candidate or prospect is deleted from Greenhouse.
         "first_name": "Johnny",
         "last_name": "Smith",
         "title": "Previous Title",
-        "external_id": 00000,
+        "external_id": 12345,
```

Comment:
External ID is actually a string value to support systems that use more complex IDs. Can you add a double quote wrapper?